src/coverlet.console/Program.cs

```diff
@@ -185,10 +186,14 @@ static int Main(string[] args)
                 coverageTable.AddRow("Average", $"{totalLinePercent / numModules}%", $"{totalBranchPercent / numModules}%", $"{totalMethodPercent / numModules}%");
 
                 logger.LogInformation(coverageTable.ToStringAlternative());
-
+                if (process.ExitCode > 0)
+                {
+                    exitCode = (int)CommandExitCodes.TestFailed;
```

Comment:
```suggestion
                    exitCode += (int)CommandExitCodes.TestFailed;
```

Reply:
I am fine with making this change. Just for my understanding. Is it because, we changed to use Flags in Enum, doing `+=` makes sense than `=` even though it be first time value is asserted based on some condition?

Reply:
Yes nothing particular, only keep the "idea" coherent.
If in future we add something above is less prone to error.